tests/js/foo.js

```diff
@@ -0,0 +1,4 @@
+
+test('Post truth', () => {
+    expect(true).toBe(true)
+})
```

Comment:
Damn, we'll have to have the talk about semicolons in JS... As you must have guessed, I'm in favor of semicolons everywhere. The reasons are numerous and many have blogged about excellent reasons why we should use them. Also, this is actually the very first time I'm seeing semicolon-less JS code in my career at Mozilla, and it was surprising. 

So, please, semicolons? :-)

Reply:
k

Reply:
we need to make sure we are linting for that

Reply:
Isn't that the case with eslint by default?

Reply:
not with current settings - i cribbed a pretty relaxed moanset